phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widget/MiniatureListWidget.java

```diff
@@ -61,7 +61,11 @@ private MiniatureListWidget(Pane content, Function<E, Element> converter, BiCons
         this.mappedElements = new MappedList<Element<E>, E>(items, value -> {
             Element newElement = converter.apply(value);
 
-            newElement.setOnMouseClicked(event -> setOnMouseClicked.accept(newElement, event));
+            newElement.setOnMouseClicked(event -> {
+                unselectAll();
+                setOnMouseClicked.accept(newElement, event);
+                select(newElement);
```

Comment:
I think it would be enough if we use a different color to mark the hovered miniature.